app/packs/src/apps/mydb/elements/details/wellplates/propertiesTab/WellplateSizeDropdown.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+import PropTypes from 'prop-types';
+import Wellplate from 'src/models/Wellplate';
+
+function newOption(wellplate, optionsKey) {
+  const size = (wellplate.width * wellplate.height).toString();
+  const width = wellplate.width.toString();
+  const height = wellplate.height.toString();
+  return {
+    value: optionsKey,
+    label: `${size} (${width}x${height})`
+  };
+}
+
+export default class WellplateSizeDropdown extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      options: [
+        { value: '24;16', label: '384 (24x16)' },
```

Comment:
@FabianMauz, are you going to address this? Otherwise I'm closing this comment as part of the second review iteration. It's not critical anyway, just a minor improvement.

Reply:
I agree with you. it's also redundant information in the options, although it's very local. I will address this.

Reply:
Resolved with c2a086173dbaefaf7078001a6e655e3ebc325931.